buddyboss/buddyboss-platform
# Uncaught TypeError: g(...).magnificPopup is not a function - fixed

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2680
Fixes #2683
Fixes #2681

### How to test the changes in this Pull Request:

1. Deactivate the moderation component
2. Browse the news feed page or any page of the website
3. See the JS error is not happening anymore


### Other information:




### Changelog entry

> Uncaught typeError JS error fixed
